web/src/pages/map/doc.vue

```diff
@@ -163,9 +173,15 @@ export default VueParent.extend({
           this.resource_title = resource_url
             ? `${resource_url.protocol}//${resource_url.host}`
             : data.resource;
+          this.item = data.item;
         }
       );
     },
+    hide_items() {
+      const checkbox = document.getElementById('checkbox-item-' + this.item);
```

Comment:
We are in Vue JS, d not access the DOM.
An event have to be send to the menu component.